packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -384,6 +388,7 @@ class ContextMenu extends MenuBase {
   }
 
   _renderContentImpl(): void {
+    this._showContextMenuEventHandler = this._createShowContextMenuEventHandler();
```

Comment:
Hm, looks like events will not be unsubscribed now because you recreate the handler before detach => link is already different when we try to off.
p.s. don't forget to add a test